templates/portal/teaching.html

```diff
@@ -0,0 +1,101 @@
+            servers throughout. In the event of a server issue, we will do all we can to resolve it quickly. We
+            recommend having a backup plan in case there are unforeseen issues that cannot be resolved promptly.</p>
+          <h3>How is SciLifeLab Serve funded?</h3>
+          <p>SciLifeLab Serve is supported by a grant from the Knut and Alice Wallenberg Foundation given to
+            SciLifeLab for the purpose of supporting research in data-driven life science in Sweden. SciLifeLab Serve
+            is available free of charge to researchers affiliated with Swedish research institutions, regardless of an
+            affiliation with SciLifeLab. It is also offered to ScLifeLab infrastructure units. Refer to our <a
+              href="{% url 'portal:about' %}">'about' page</a> for more information.</p>
+
+        <h2 id="application">Application form</h2>
+
+        <div class="container">
+            <form class="needs-validation" action="https://forms.dckube.scilifelab.se/api/v1/form/noRF3FO03F_2F8pL/incoming" method="POST" accept-charset="utf-8">
+            <input type='hidden' name='csrfmiddlewaretoken' value='{{ csrf_token }}' />
+            <div class="row justify-content-center">
+              <div class="col-6 border rounded p-4">
+                <div class="mb-3">
+                  <label for="name" class="form-label">Your name:</label>
+                  <input type="text" class="form-control" name="name" id="name">
+                </div>
+                <div class="mb-3">
+                  <label for="email" class="form-label">Your email address(that connected to your SciLifeLab Serve
+                    account):</label>
+                  <input type="email" class="form-control" name="email" id="email">
+                </div>
+                <div class="mb-3">
+                  <label for="course_title" class="form-label">Title of course/workshop/webinar:</label>
+                  <input type="text" class="form-control" name="course_title" id="course_title">
+                </div>
+                <div class="mb-3">
+                  <label for="course_dates" class="form-label">Date(s) and time(s) of couse/workshop/webinar:</label>
+                  <input type="text" class="form-control" name="course_dates" id="course_dates">
+                </div>
+                <div class="mb-3">
+                  <label for="course_description" class="form-label">Description of couse/workshop/webinar:</label>
+                  <textarea class="form-control" name="course_description" id="course_description" rows="4"></textarea>
+                </div>
+                <div class="mb-3">
+                  <button class="btn btn-primary" type="submit">Submit application</button>
+                </div>
+                <div class="mb-3">
+                  <div class="g-recaptcha" data-sitekey="6LfX3uggAAAAAFOoQPUE-iPY8wR9dPYp7bIBTc9W"></div>
+                </div>
```

Comment:
The captcha displays `ERROR for site owner:
Invalid domain for site key`, but i guess this is because i run it on my localhost?

Reply:
Yes, unfortunately can only be tested on either dev or production.